nrf_wifi/fw_if/umac_if/inc/fmac_structs_common.h

```diff
@@ -38,6 +38,17 @@ struct rpu_host_stats {
 	unsigned long long total_rx_drop_pkts;
 };
 
+/**
+ * @brief Structure to hold firmware debug statistics.
+ *
+ */
+struct rpu_fw_debug_stats{
+	/** RPU hardware lockup event detection count */
+	unsigned int rpu_hw_lockup_count;
+	/** RPU hardware lockup recovery completed count */
+	unsigned int rpu_hw_lockup_recovery_done;
+};
```

Comment:
Why does this structure need to be defined in the driver header file ? Shouldn't this be picked up from the FW shared header file?

Reply:
There is no structure available to store the value. They have defined the variables with event structure. So need to store the value.